spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/integration/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2012-2018 the original author or authors.
```

Comment:
Is it really OK to have Copyright in the `package-info.java` files?

Reply:
Yes please, that's consistent with the others that we have.